src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,23 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User newUser = new UserService().findByEmail(email);
+        if (newUser != null) {
+            return newUser.getPassword().equals(password);
+        }
         return false;
```

Comment:
Мы возвращаем `boolean`, то-есть ответ будет `true` только если выполнятся условия `newUser != null` и `newUser.getPassword().equals(password)`, поэтому можем объединить:
```suggestion
        return newUser != null && newUser.getPassword().equals(password);
```